cmd/jaeger/config-kafka-collector.yaml

```diff
@@ -40,7 +40,7 @@ processors:
 exporters:
   kafka:
     brokers:
-      - localhost:9092
+      - ${env:KAFKA_BROKERS:-localhost:9092}
```

Comment:
this is a single broker. If you want to support multiple brokers then I'd expect those to be provided as array, something like this
```
brokers: [ ${env:KAFKA_BROKERS:-localhost:9092} ]
```
But I would just keep it as is as a singular KAFKA_BROKER